### PR TITLE
Add consolidate release notes for 2.0.1

### PR DESCRIPTION
**Pull request by zelinh**
Signed-off-by: Zelin Hao <zelinhao@amazon.com>

### Description
Add consolidate release notes for 2.0.1

### Issues Resolved
Part of #2165 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by lezzago — APPROVED** [review]

**Review by dreamer-89 — APPROVED** [review]
LGTM!

**Review by kotwanikunal — APPROVED** [review]

File: release-notes/opensearch-release-notes-2.0.1.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+### OpenSearch Alerting Dashboards Plugin
+* Implemented a fix for issue 258 which was allowing the UX to define more than 1 index for document level monitors ([#259](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/259))
+* Fixed a bug that was causing the action execution policy to be configurable for query and cluster metrics monitors ([#261](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/261))
+* Fixed an issue preventing doc level monitors from adding execution policy as expected ([#262](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/262))
+* Fixed an issue that would sometimes cause the loadDestinations function to not call getChannels ([#264](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/264))
+
+### Opensearch Anomaly Detection
+* Use current time as training data end time ([#547](https://github.com/opensearch-project/anomaly-detection/pull/547))
+* bump rcf to 3.0-rc3 ([#568](https://github.com/opensearch-project/anomaly-detection/pull/568))
```
Comment (Naarcha-AWS) [inline release-notes/opensearch-release-notes-2.0.1.md:21]: Capitalize "Bump".

File: release-notes/opensearch-release-notes-2.0.1.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### Maintenance
+
+### OpenSearch Alerting Dashboards Plugin
+* Incremented version to 2.0.1 ([#269](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/269))
+
+### Documentation
```
Comment (Naarcha-AWS) [inline release-notes/opensearch-release-notes-2.0.1.md:28]: Let's remove the Documentation section if there are no specific updates.

File: release-notes/opensearch-release-notes-2.0.1.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# OpenSearch and OpenSearch Dashboards 2.0.1 Release Notes
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 2.0.1 includes the following bug fixes, documentation, and maintenance updates.
```
Comment (Naarcha-AWS) [inline release-notes/opensearch-release-notes-2.0.1.md:5]: Remove "documentation" from this sentence.

Comment (zelinh) [reply release-notes/opensearch-release-notes-2.0.1.md:5]: Actually there is one from alerting dashboards. Changed the font size for the `Documentation` below.

**Review by Naarcha-AWS — APPROVED** [review]

**Pull request merged**